src/gui/accountmanager.h

```diff
@@ -58,7 +58,7 @@ class AccountManager : public QObject
      * Return a list of all accounts.
      * (this is a list of QSharedPointer for internal reasons, one should normally not keep a copy of them)
      */
-    QList<AccountStatePtr> accounts() { return _accounts; }
+    const QList<AccountStatePtr> &accounts() { return _accounts; }
```

Comment:
This is only ever used from a single thread, right?

Reply:
I seriously hope so....